src/bodywork/git.py

```diff
@@ -240,3 +242,7 @@ def get_git_commit_hash(project_path: Optional[str] = None) -> str:
         raise BodyworkGitError(
             f"Unable to retrieve git commit hash: {e.stdout} {e.stderr}"
         ) from e
+    except OSError as e:
```

Comment:
Can we have a test for this path?

Reply:
I'm guessing this is for the sake of the CodeCov report.

Reply:
Done

Reply:
Nice one thanks.